### PR TITLE
Changes done to NSJSONSerialization.swift to align with darwin code

### PR DESCRIPTION
**Pull request by saiHemak**

### CODE REVIEW

File: TestFoundation/TestNSJSONSerialization.swift [hunk 1]
```diff
@@ -75,7 +75,7 @@ extension TestNSJSONSerialization {
         ]
         
         for (description, encoded) in subjects {
-            let result = try? NSJSONSerialization.JSONObjectWithData(NSData(bytes:UnsafePointer<Void>(encoded), length: encoded.count), options: [])
+            let result = try? NSJSONSerialization.jsonObject(with:NSData(bytes:UnsafePointer<Void>(encoded), length: encoded.count), options: [])
```
Comment (ianpartridge) [inline TestFoundation/TestNSJSONSerialization.swift]: I think codingstyle is for there to be a space after the colon.

**Comment by parkera** [comment]
Agreed with @ianpartridge on the style note.

**Comment by saiHemak** [comment]
Sorry I missed  it ..I corrected it ..

**Comment by saiHemak** [comment]
@ianpartridge  I have squashed the three commits ..Thanks !!

**Comment by parkera** [comment]
@swift-ci please test

**Pull request merged**